data/hydrophobic_res/hydrophobic_res.py

```diff
@@ -0,0 +1,7 @@
+from mdtraj import load_pdb
+import os
+t = load_pdb(os.path.join(_FILE_DIR_,'e22g.pdb'))
+top = t.topology
+hp_res_list = ['ALA', 'VAL', 'LEU', 'ILE', 'PRO', 'PHE', 'CYS']
+hydrophobic_res = [res for res in top.residues if str(res)[:3] in hp_res_list]
+result = True if get_hydrophobic_residues(os.path.join(_FILE_DIR_,'e22g.pdb')) == hydrophobic_res else False
```

Comment:
LGTM. The exceptions are because the response length needs to be increased. Think the prompt is fine.

Have you tested this prompt? I'm not sure what kind of object `res` is

Reply:
Yes, I have tested this and it works well. `res` is a `<class 'mdtraj.core.topology.Residue'>` object, hence has to convert to string.

Reply:
How do you suggest we increase prompt response length? Is that in the nlcc code or needs to be done in the `.yml` file?

Reply:
No need, you're fine. I'll approve